via/services/youtube.py

```diff
@@ -67,9 +67,14 @@ def get_video_id(self, url):
 
     def get_video_title(self, video_id):
         """Call the YouTube API and return the title for the given video_id."""
-        # https://developers.google.com/youtube/v3/docs/videos/list
+        video = self._db.scalar(select(Video).where(Video.video_id == video_id))
+
+        if video:
```

Comment:
I guess is now idiomatic python to use `if video :=` here

Reply:
:+1: Nice! Done